.github/workflows/ci.yml

```diff
@@ -6,6 +6,12 @@ on: [push, pull_request]
 env:
   EXPORT_DIR: exported-artifacts
 jobs:
+  copyright:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: REUSE Compliance Check
+      uses: fsfe/reuse-action@v1
```

Comment:
This does not work locally, only on github, and it requires a separate job for lint that take only few seconds. Better to add reuse to our standard lint target.

Reply:
Ok, I have explored both options, so I will recover that change. It will require to add `reuse` to the requirement.txt and update the containers.

Reply:
Adding to the containers is little more work but we need to update them anyway, and we really want to have a working local lint for quick local checks.